development-utility/development-server/src/com/thoughtworks/go/server/DevelopmentServer.java

```diff
@@ -39,8 +40,9 @@
 
 public class DevelopmentServer {
     public static void main(String[] args) throws Exception {
+        LogConfigurator logConfigurator = new LogConfigurator(DEFAULT_LOG4J_CONFIGURATION_FILE);
+        logConfigurator.initialize();
```

Comment:
This happens because the main program uses a different classloader than the webapp. So the logger is actually initialized twice. Side-effect of how logging is setup, unfortunately nothing (simple) that can be done about it.